### PR TITLE
adding PU 8400 material file

### PR DESCRIPTION
**Pull request by Mick3Lozzo**
Adding PU 8400 mtl file

Data from datasheets.

[Hei-Cast PU 8400.pdf](https://github.com/user-attachments/files/17800485/Hei-Cast.PU.8400.pdf)
[PU8400 Rubber like.pdf](https://github.com/user-attachments/files/17800488/PU8400.Rubber.like.pdf)
[PU 8400 Xometry.pdf](https://github.com/user-attachments/files/17800494/PU.8400.Xometry.pdf)

### CODE REVIEW

File: config/materials/hei-cast_pu-8400.mtl [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  Name = PTC_MATERIAL_SUB_TYPE
+  Type = String
+  Default = 'LINEAR'
+  Access = Locked
+},
+{
+  Name = PTC_MATERIAL_DESCRIPTION
+  Type = String
+  Default = 'Polyurethane resin -vacuum casting urethane for prototypes
+shore 70'
```
Comment (fiorisi) [inline config/materials/hei-cast_pu-8400.mtl:34]: @Mick3Lozzo the shore hardness for this material can range between 60 and 90. Therefore, we should either remove this property from the description or differentiate the name based on the shore hardness.

**Review by fiorisi — CHANGES_REQUESTED** [review]

**Comment by Mick3Lozzo** [comment]
With @Lawproto we decided to rename the file to identify unmistakably the material.

File: config/materials/hei-cast_pu-8400_shore-70.mtl [hunk 2]
```diff
@@ -0,0 +1,68 @@
+﻿ND_RelParSet_K01 = {
+
+Name = HEI-CAST_PU-8400
```
Comment (Lawproto) [inline config/materials/hei-cast_pu-8400_shore-70.mtl:3]: I think this should match the file name

**Review by Lawproto — CHANGES_REQUESTED** [review]

File: config/materials/hei-cast_pu-8400_shore-70.mtl (hunk 2 above)
Comment (Mick3Lozzo) [reply config/materials/hei-cast_pu-8400_shore-70.mtl]: Done.

**Review by Lawproto — APPROVED** [review]

**Review by fiorisi — APPROVED** [review]

**Pull request merged**